extensions/presentation.sigmf-ext.md

```diff
@@ -0,0 +1,239 @@
+# The `presentation` SigMF Extension Namespace v1.0.0
+
+This document defines the `presentation` extension namespace for the Signal
+Metadata Format (SigMF) specification. This extension namespace contains objects
+to help define how SigMF metadata should be presented graphically to users.
+
+All fields in the `presentation` extension are optional, thus any SigMF
+application using this extension is not required to implement any particular
+feature. It is RECOMMENDED that applications adhering to the `presentation`
+extension use default values in this document. It is also RECOMMENDED that
+applications implementing this extension describe what fields are implemented.
+
+## 0 Datatypes
+
+The `presentation` extension defines the following SigMF datatypes:
+
+|type|long-form name|description|
+|----|--------------|-----------|
+|`color`|Hexadecimal Color String|String representing a 24 bit color (with optional alpha value) in hexadecimal form; either "#RRGGBB" or "#AARRGGBB".|
+
+### 0.1 The `color` datatype
+
+The `color` datatype is a string which MUST be either 6 or 8 hexadecimal
```

Comment:
Did you consider also allowing strings from this well-defined set of color names? https://www.w3.org/TR/SVG11/types.html#ColorKeywords

Reply:
I think this would be ver nice overall but I am not sure if this is burdensome on applications implementing this. Interestingly the merge request with inspectrum does support this (thanks to QT's native support for it).

Open to any thoughts here, especially situations this would be a problem.

Reply:
Another thought (I'm not strongly advocating this, but I do like it) is to separate the color and the alpha into two separate fields.  That would eliminate the RGBA vs ARGB issue, and simultaneously allow for alpha-izing named colors.

Inasmuch as the `.sigmf-meta` files are supposed to be "human readable", and inasmuch as I still have a hard time visually decoding hex color strings, named colors would be ... more understandable.

Reply:
human readability is a strongly beneficial reason to do this. Not sold on separate alpha but i see your use case...will think more. could be an optional that overrides AARRGGBB....